app/livechat/server/api/v1/config.js

```diff
@@ -10,16 +10,18 @@ API.v1.addRoute('livechat/config', {
 			check(this.queryParams, {
 				token: Match.Maybe(String),
 				department: Match.Maybe(String),
+				businessUnit: Match.Maybe(String),
 			});
 			const enabled = Livechat.enabled();
 
 			if (!enabled) {
 				return API.v1.success({ config: { enabled: false } });
 			}
 
-			const config = Promise.await(settings());
+			const { token, department, businessUnit } = this.queryParams;
+
+			const config = Promise.await(settings({ businessUnit }));
```

Comment:
You can make the `get` async now, and use normal await instead of this one 👀 
```suggestion
			const config = await settings({ businessUnit });
```